test/runner/package.json

```diff
@@ -0,0 +1,3 @@
+{
+  "type": "module"
```

Comment:
Can we put that at the top level as in Core? `Gruntfile.js` can be renamed to `Gruntfile.cjs` for now.